ui/src/translations/fr.json

```diff
@@ -141,7 +141,8 @@
   "env_preparation": "Préparation de l’environnement",
   "env_preparation_success": "L'environnement {envName} est en cours de préparation.",
   "env_preparation_failed": "La préparation de l'environnement {envName} a échoué. {step} - {reason}",
-  "preparing_environemnt": "Préparer l'environnement",
+  "preparing_environment": "Préparer l'environnement",
```

Comment:
If we have both "prepare" and "preparing", the right translation for "preparing" would be `Environnement en cours de préparation`.
What bugs me is that `env_preparation_success` already means "in progress", for some reason. Should we make sure all those translations are used appropriately?

Reply:
Thanks for the french translation of "preparing"!

Well, actually these two keywords hold the same meaning indeed, but they are used by different components.

`"env_preparation_success": "Environment {envName} is being prepared." `used by the notification 
`"preparing_environment": "Preparing the environment" `used by the loader in the env list when adding a solution which should be a shorter version.

so for me, it may not clear which one to use but make sure to have both.